Makefile

```diff
@@ -24,4 +24,6 @@ irel: $(GO_SOURCES)
 release: $(GO_SOURCES) test
 	GOOS=darwin   GOARCH=amd64 go build -o $(OUTPUT)     cmd/irel/main.go && tar -czf irel-darwin-amd64.tgz  $(OUTPUT)     && rm -f $(OUTPUT)
 	GOOS=linux    GOARCH=amd64 go build -o $(OUTPUT)     cmd/irel/main.go && tar -czf irel-linux-amd64.tgz   $(OUTPUT)     && rm -f $(OUTPUT)
-	GOOS=windows  GOARCH=amd64 go build -o $(OUTPUT).exe cmd/irel/main.go && zip -mq  irel-windows-amd64.zip $(OUTPUT).exe && rm -f $(OUTPUT).exe
+	#GOOS=windows  GOARCH=amd64 go build -o $(OUTPUT).exe cmd/irel/main.go && zip -mq  irel-windows-amd64.zip $(OUTPUT).exe && rm -f $(OUTPUT).exe
```

Comment:
one advantage of not running in a container is that you can install bits into the vm and have them be available to later stages

Reply:
True. I'll see if that approach can be made to work.